components/molecule/photoUploader/src/DragState.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const DRAG_STATE_CLASS_NAME = `${BASE_CLASS_NAME}-dragState`
+const DEFATUL_DRAG_STATE_STATUS = 'accepted'
+
+const DragState = ({icon, status = DEFATUL_DRAG_STATE_STATUS, text}) => {
+  const dropzoneClassName = cx(DRAG_STATE_CLASS_NAME, {
+    [`${DRAG_STATE_CLASS_NAME}--accepted`]: status === 'accepted',
+    [`${DRAG_STATE_CLASS_NAME}--rejected`]: status === 'rejected'
```

Comment:
Does it make sense to replace 'rejected' to DRAG_STATE_STATUS_REJECTED?

Reply:
same here, with:
```
[`${DRAG_STATE_CLASS_NAME}--${status}`],
```